### PR TITLE
[r] Add comp_bio_census_info.Rmd

### PR DESCRIPTION
**Pull request by mlin**
Furthering https://github.com/chanzuckerberg/cellxgene-census/issues/248

This had been blocked for awhile on: https://github.com/single-cell-data/TileDB-SOMA/issues/1262 now fixed

PREVIEW: https://gistpreview.github.io/?85f31f2c190c4fa4e99edbebc0eae15c

### CODE REVIEW

**Comment by mlin** [comment]
@pablo-gar The GitHub Actions worker may be running out of memory on the new vignette. I think they have only 7GB of RAM. Two possible courses of action (1) scale down the problem size in the vignette until it runs successfully, or (2) dispense with automatically building [all of] the vignettes in the CI workflow, which (correct me?) I don't think we do with the Python notebooks. Lets talk about which way to go.

File: api/r/cellxgene.census/vignettes/comp_bio_census_info.Rmd [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+- Opening the census
+- Census organization
+- Cell metadata
+- Gene metadata
+- Census summary content tables
+- Understanding Census contents beyond the summary tables
+
+## Opening the Census
+
+The `cellxgene.census` R package  contains a convenient `open_soma()` API to open any version of the Census (the latest by default).
```
Comment (ebezzi) [inline api/r/cellxgene.census/vignettes/comp_bio_census_info.Rmd:31]: I believe we changed this to `stable` by default, at least in python. Has this been done in R as well @atolopko-czi ? If not, we should probably uniform the behavior and update this vignette.

File: api/r/cellxgene.census/vignettes/comp_bio_census_info.Rmd [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+
+```{r}
+census <- cellxgene.census::open_soma()
+```
+
+You can learn more about the cellxgene.census methods by accessing their corresponding documentation, for example `?cellxgene.census::open_soma`.
+
+## Census organization
+
+The [Census schema](https://cellxgene-census.readthedocs.io/en/latest/schema.html) defines the structure of the Census. In short, you can think of the Census as a structured collection of items that stores different pieces of information. All of these items and the parent collection are SOMA objects of various types and can all be accessed with the [TileDB-SOMA API](https://github.com/single-cell-data/TileDB-SOMA) ([documentation](https://tiledbsoma.readthedocs.io/en/latest/)).
```
Comment (ebezzi) [inline api/r/cellxgene.census/vignettes/comp_bio_census_info.Rmd:41]:
```suggestion
The [Census schema](https://chanzuckerberg.github.io/cellxgene-census/cellxgene_census_docsite_schema.html) defines the structure of the Census. In short, you can think of the Census as a structured collection of items that stores different pieces of information. All of these items and the parent collection are SOMA objects of various types and can all be accessed with the [TileDB-SOMA API](https://github.com/single-cell-data/TileDB-SOMA) ([documentation](https://tiledbsoma.readthedocs.io/en/latest/)).
```

File: api/r/cellxgene.census/vignettes/comp_bio_census_info.Rmd [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+
+The `cellxgene.census` package contains some convenient wrappers of the TileDB-SOMA API. An example of this is the function we used to open the Census: `cellxgene_census.open_soma()`.
+
+### Main Census components
+
+With the command above you created `census`, which is a `SOMACollection`, an R6 class supplying providing a key-value associative map. Its `get()` method can access the two top-level collection members, `census_info` and `census_data`, each themselves instances of `SOMACollection`.
```
Comment (ebezzi) [inline api/r/cellxgene.census/vignettes/comp_bio_census_info.Rmd:47]:
```suggestion
With the command above you created `census`, which is a `SOMACollection`, an R6 class providing a key-value associative map. Its `get()` method can access the two top-level collection members, `census_info` and `census_data`, each themselves instances of `SOMACollection`.
```

File: api/r/cellxgene.census/vignettes/comp_bio_census_info.Rmd [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+
+## Census summary content tables
+
+You can take a quick look at the high-level Census information by looking at `census$get("census_info")$get("summary")`:
+
+```{r}
+as.data.frame(census$get("census_info")$get("summary")$read())
+```
+
+Of special interest are the label-value combinations for :
```
Comment (ebezzi) [inline api/r/cellxgene.census/vignettes/comp_bio_census_info.Rmd:106]:
```suggestion
Of special interest are the label-value combinations for:
```

**Review by ebezzi — CHANGES_REQUESTED** [review]

File: api/r/cellxgene.census/vignettes/comp_bio_census_info.Rmd (hunk 1 above)
Comment (atolopko-czi) [reply api/r/cellxgene.census/vignettes/comp_bio_census_info.Rmd:31]: Nope, good call! Added https://github.com/chanzuckerberg/cellxgene-census/issues/482

Comment (ebezzi) [reply api/r/cellxgene.census/vignettes/comp_bio_census_info.Rmd:31]: Thanks Andrew! In this case, I will resolve this conversation as we should tackle that in a separate PR. I added a note in the ticket to update the vignette.

Comment (mlin) [reply api/r/cellxgene.census/vignettes/comp_bio_census_info.Rmd:31]: Agree- I'll take care of #482 once the `stable` key exists in the `release.json`!

**Comment by mlin** [comment]
@ebezzi @pablo-gar FYI- I've usurped this old PR to also update the vignettes to the recent breaking changes, and move them into `vignettes_wip/` temporarily to hopefully unblock the docsite and r-universe builds.

**Pull request merged**